lambdas/tabular_preview/requirements.txt

```diff
@@ -1,56 +1,150 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
you should use the same Python version we use in production i.e. 3.11
even better is to migrate to uv.lock in a separate prefactoring PR